contrib/build-container.sh

```
@@ -157,6 +165,14 @@ get_options() {
                 missing_requirement $1
             fi
             ;;
+        --wheel-base-image)
+            if [ "$2" ]; then
+                WHEEL_BASE_IMAGE=$2
+                shift
+            else
+                missing_requirement $1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)